spec/features/bulk_update_metadata_spec.rb

```diff
@@ -6,34 +6,34 @@
   let(:note) { RandomWord.phrases.next }
 
   before do
-    authenticate!(start_url: BASE_URL, expected_text: 'Welcome to Argo!')
+    authenticate!(start_url: Settings.argo_url, expected_text: 'Welcome to Argo!')
   end
 
   scenario do
     druid1 = create_druid
     druid2 = create_druid
     temp_xlsx = update_xlsx(druid1, title1, druid2, title2)
 
-    visit("#{BASE_URL}view/#{APO}")
-    # Opens the MODS bulk jobs
+    visit "#{Settings.argo_url}/view/#{Settings.default_apo}"
+    # Open the MODS bulk jobs
     click_link 'MODS bulk loads'
     expect(page).to have_content 'Datastream spreadsheet bulk upload for APO'
 
-    # Opens the Submit new file modal
+    # Open the Submit new file modal
     click_link 'Submit new file ...'
     expect(page).to have_content 'Submit MODS descriptive metadata for bulk processing'
 
-    # Attaches spreadsheet fixture, selects spreadsheet input, and adds note
+    # Attach spreadsheet fixture, select spreadsheet input, and add note
     attach_file('Select', temp_xlsx.path)
     choose 'Spreadsheet input; load into objects'
     fill_in '3. Note', with: note
     click_button 'Submit'
 
     reload_page_until_timeout!(text: note)
 
-    # Delete's job run
+    # Delete job run
     job_rows = page.find_all('div#bulk-upload-table > table > tbody > tr')
-    job_rows.drop(1).each do |row|
```

Comment:
why was this dropping a row?  Why don't we need to any more?

Reply:
No need to drop a row since the check on line 38 is going to skip the header row anyway. It's noise, IMO.